lib/cli.ex

```diff
@@ -59,11 +60,18 @@ defmodule Gradient.CLI do
     underscore_color: :string,
     # path and compiler options
     path_add: :string,
-    module: :string
+    module: :string,
+    help: :boolean
   ]
 
   def main(args) do
     {options, user_paths, _invalid} = OptionParser.parse(args, strict: @options)
+    help_flag_set? = Keyword.get(options, :help, false)
+
+    if (length(options) == 0 and length(user_paths) == 0) or help_flag_set? do
+      IO.puts(@moduledoc)
```

Comment:
```suggestion
      IO.puts(@moduledoc |> (&Regex.replace(~r/(  \* |`)/, &1, "")).()
```

Please bring back the `@moduledoc` formatting - we can strip it like this if need be. We want nice docs in the future, so let's think of it already today.